src/DIRAC/FrameworkSystem/scripts/dirac_admin_update_pilot.py

```diff
@@ -38,21 +39,24 @@ def updatePilot(version, vo):
         :param version: version vArBpC of pilot you want to use
         :param vo: Location of pilot version in CS /Operations/<vo>/Pilot/Version
         """
-        # FIXME: use Operations() object
-        pilotVersion = gConfig.getValue(f"Operations/{vo}/Pilot/Version", [])
-        if not pilotVersion:
-            return S_ERROR(f"No pilot version set under Operations/{vo}/Pilot/Version in CS")
-
-        pilotVersion.pop()
-        pilotVersion.insert(0, version)
+        res = Operations().getValue("Pilot/Version", [])
```

Comment:
don't you need  to pass `vo` as parameter to `Operations`? At least, this is what @andresailer always told me to :nerd_face:

Reply:
ah yes! corrected.